WeScan/Scan/ScannerViewController.swift

```diff
@@ -79,6 +81,10 @@ final class ScannerViewController: UIViewController {
         setupNavigationBar()
         setupConstraints()
         
+        CaptureSession.current.isAutoScanEnabled.toggle() //to set right value when the below toggleAutoScan is trigger
```

Comment:
Why are we toggling the `isAutoScanEnabled` when it previously already had an intended default value? This could change the expected behavior - if we really want to do it we need to mark this as a breaking change.